sim/core/aura.go

```diff
@@ -286,24 +288,16 @@ func (unit *Unit) GetOrRegisterAura(aura *Aura) *Aura {
 }
 
 func (at *auraTracker) GetAurasWithTag(tag string) []*Aura {
-	if auras, ok := at.aurasByTag[tag]; ok {
-		return auras
-	} else {
-		return []*Aura{}
-	}
+	return at.aurasByTag[tag]
 }
 
-func (at *auraTracker) GetActiveAurasWithTag(tag string) []*Aura {
-	active := []*Aura{}
-	for _, aura := range at.GetAurasWithTag(tag) {
-		if aura.IsActive() {
-			active = append(active, aura)
+func (at *auraTracker) HasActiveAuraWithTag(tag string) bool {
+	for _, aura := range at.aurasByTag[tag] {
+		if aura.active {
+			return true
 		}
 	}
-	return active
-}
-func (at *auraTracker) HasActiveAuraWithTag(tag string) bool {
-	return len(at.GetActiveAurasWithTag(tag)) > 0
+	return false
 }
```

Comment:
GetActiveAurasWithTag() was only used for HasActiveAuraWithTag(), so I cut the middleman. I don't think the tag support is really needed, but it's probably easier to use.

Reply:
If you have another idea for how to manage groups of auras, a PR would be welcome.

Reply:
If tagging is only used for inter-aura priority management (IEA overwriting SA etc.), I think that could be handled like before. I haven't looked into other use cases, though, and grouping might help in other scenarios.